embark.el

```diff
@@ -670,6 +678,14 @@ If CYCLE is non-nil bind `embark-cycle'."
    (symbol-value (or (alist-get type embark-keymap-alist)
                      (alist-get t embark-keymap-alist)))))
 
+(defun embark--truncate-target (target)
+  "Truncate TARGET string."
+  (unless (stringp target)
+    (setq target (format "%s" target)))
+  (if-let (pos (string-match-p "\n" target))
+      (concat (substring target 0 pos) "…")
+    target))
+
```

Comment:
this is a good idea, yes.  for cases where emacs runs maximased
(for instance, i have 270 columns at my disposal) one could consider
joining lines (e.g., to show a full sexp) up until (window-width), but
i am not sure about the result.  this simpler approach is probably better.

Reply:
Yes, for now I've avoided using a more complicated truncation method. Truncation is a very recent addition so we I am certain we will improve this further at some point.